Symfony/CS/FileCacheManager.php

```diff
@@ -86,6 +90,11 @@ private function calcHash($content)
         return crc32($content);
     }
 
+    private function getRelativePath($file)
+    {
+        return str_replace($this->dir, './', $file);
```

Comment:
@Slamdunk 

This is already taken care of, see `Directory::getRelativePathTo()`. 

As it stands without the most recent commit, even when passing in the absolute path to the `FileCacheManager`, paths relative to the cache file are stored within the cache file.